### ISSUE TITLE
请教gcn中的实现

### ISSUE THREAD

**Issue by huzhanchi** [alibaba/graph-learn#117]
你好，gl中的gcn实现有些不太明白，看到是把`updated_vecs = tf.reduce_sum([self_vecs, neigh_vecs], axis=0)`丢进densenet。没见到原参考论文 [gcn](https://arxiv.org/pdf/1609.02907v4.pdf) 中的 拉普拉斯矩阵归一化的处理。如有时间帮忙解惑下，感激！

**Comment by baoleai** [comment]
你好，这里的确没有归一化处理，归一化处理需要获取点的度来进行归一化，这个我们后续会支持的。

2020-11-24 16:35 GMT+08:00, huzhanchi <notifications@github.com>:

**Comment by huzhanchi** [comment]
期待！再请教一下，现在gcn代码里这种K阶邻居的聚合方式有什么文献可以了解的么？

**Comment by huzhanchi** [comment]
感谢你的回答，等待新功能
